docs/get_started.rst

```diff
@@ -1,17 +1,388 @@
+            namespace: sampleNamespace
+        .
+        .
+        .
+
+The ``cumulusci.yml`` file configures your project-specific tasks, flows, and customizations.
+
+For more information regarding configuration, check out our `project configuration <#TODO internal ref here>`_ section of the docs.
+You can even add and commit it to your Git repository:
+
+.. code-block:: console
+
+    $ git add cumulusci.yml
+    $ git commit -m "Initialized CumulusCI Configuration"
+
+
+
+Add Your Repo to GitHub
+^^^^^^^^^^^^^^^^^^^^^^^
+
+With your ``cumulusci.yml`` file committed, create a repository on GitHub for your new project and push your changes there.
+There are multiple ways to do this:
+
+* Our `Community Project Development with CumulusCI <https://trailhead.salesforce.com/content/learn/modules/community-project-development-with-cumulusci?trail_id=build-applications-with-cumulusci>`_ module covers GitHub Desktop in the "Set Up the Project" section.
+* If you prefer the command line, GitHub has excellent guides on both `git remote <https://github.com/git-guides/git-remote>`_ and `git push <https://github.com/git-guides/git-push>`_.
+
+
+
+Convert an Existing Salesforce Project
+--------------------------------------
+
+Converting an existing Salesforce project to use CumulusCI may follow a number of different paths, depending on whether you're practicing the Org Development Model or the Package Development Model; whether or not you're already developing in scratch orgs; and the complexity of your project's dependencies on the org environment.
+
+If you're developing in persistent orgs and not already using source control, you'll also need to retrieve the existing metadata from the org. Note that adopting source control and moving from persistent orgs into scratch orgs are case-specific processes; your experience may vary and you may need to undertake additional work as part of those transitions.
+
+You're welcome to discuss project conversion in the `CumulusCI (CCI) Trailblazer group <https://trailblazers.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F9300000009M9Z>`_.
+
+
+
+Project Setup
+^^^^^^^^^^^^^
+
+Prior to retrieving your project's metadata you need to set up the project's root directory. These steps include:
+
+* Creating a project directory and initializing it as a Git repository as outlined in `starting a new CumulusCI project`_.
+* Initializing the directory as a CumulusCI project as outlined in `project initialization`_.
+
+
+
+Retrieving Metadata
+^^^^^^^^^^^^^^^^^^^
+
+The following assumes that your project currently lives in a persistent org, such as a Developer Edition org or a sandbox. We recommend retrieving metadata via the Metadata API (using the Salesforce CLI), followed by converting the source from Metadata API to SFDX format.
+
+#. If the metadata you want to retrieve isn't already in a package, `create one <https://help.salesforce.com/articleView?id=creating_packages.htm>`_.
```

Comment:
For accessibility reasons, it's better to make the hyperlinked text the actual title of the article that you're linking to. In this case, probably best to carve out a separate sentence. See [Create a Package](https://help.salesforce.com/articleView?id=creating_packages.htm&type=5) in Salesforce Help.